roles/gateway_update_otk_hostnames/vars/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# New value for Gateway cluster.hostname cluster property. Only updated if set to non-empty string.
+new_cluster_hostname: "" 
+
+# Values for replacing all old OTK JDBC URLs' hostnames with the new one. URLs are only updated if both are set to non-empty strings.
+old_otk_jdbc_hostname: ""
+new_otk_jdbc_hostname: ""
+
+# Values for updating all old OTK callback URLs' hostnames with the new one. URLs are only updated if both are set to non-empty strings.
+old_callback_hostname: ""
+new_callback_hostname: ""
+
+# SQL commands used by this role.
```

Comment:
I don't know what is the likely hood that the customer would need to change the SQL statements.  If not, perhaps they could be moved inside the task and use facts instead.

Reply:
Left it as the original author wrote it but I agree... it is also easier to read the tasks when the sql is in the same task file instead of reading from a separate file. Updated it.